sdk/trace/src/main/java/io/opentelemetry/sdk/trace/data/SpanData.java

```diff
@@ -50,13 +50,36 @@
    */
   TraceState getTraceState();
 
+  /**
+   * Returns the parent {@link SpanContext}. If the span is a root span, the {@link SpanContext}
+   * returned will be invalid.
+   */
+  SpanContext getParentSpanContext();
+
   /**
    * Returns the parent {@code SpanId}. If the {@code Span} is a root {@code Span}, the SpanId
    * returned will be invalid..
    *
    * @return the parent {@code SpanId} or an invalid SpanId if this is a root {@code Span}.
+   * @deprecated Use {@link #getParentSpanContext()}
    */
-  String getParentSpanId();
+  @Deprecated
```

Comment:
I don't know if we even want to deprecate that. So far, it seems exporters have only needed this, so it seems OK to keep it as convenience shortcut.
I'm fine with deprecating/removing it though, just a thought.

Reply:
I'm fine either way.

Reply:
Makes sense to me to keep it so undeprecated